### PR TITLE
1st draft nft-storage

### PR DESCRIPTION
**Pull request by DennisDawson**

### CODE REVIEW

File: content/concepts/tokens/nft-storage.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+If your data is smaller than 256 bytes, you can consider using a `data://` URI and embedding it directly in the URI field. This has the advantage of storing the data in a reliable, persistent, and responsive database.
+
+## Decentralized, off XRPL
+
+You can use any of the existing decentralized storage solutions for your NFT metadata.
+
+IPFS and Arweave offer solutions for decentralization. However, fetching the metadata efficiently can be a problem. Querying IPFS or Arweave directly to fetch metadata is not fast enough for modern websites that require immediate responses from users that are scrolling through multiple pages of NFTs that include high-quality media.
+
+For a list of cloud storage solutions, the blog post [NFT Payload Storage](https://xrpl.org/blog/2023/nft-payload-storage.html).
```
Comment (amarantha-k) [inline content/concepts/tokens/nft-storage.md]:
```suggestion
See the blog post [NFT Payload Storage](https://xrpl.org/blog/2023/nft-payload-storage.html) for some examples of cloud storage solutions.
```

File: content/concepts/tokens/nft-storage.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+html: nft-storage.html
+parent: non-fungible-tokens.html
+blurb: Storage options for the payload of your NFT.
+labels:
+  - Non-fungible Tokens, NFTs
+---
+# NFT Payload Storage
+
+NFTs are created on the block chain. But the payload of the NFT, including media, metadata, and attributes can be stored in a variety of ways, including on-chain, decentralized off-chain, and centralized off-chain.
```
Comment (amarantha-k) [inline content/concepts/tokens/nft-storage.md]:
```suggestion
NFTs are created on the blockchain. But the payload of the NFT, including media, metadata, and attributes can be stored in a variety of ways, including on-chain, decentralized off-chain, and centralized off-chain.
```

Comment (amarantha-k) [reply content/concepts/tokens/nft-storage.md]: The term "off-chain" doesn't appear to have been used in the XRPL documentation so far; "on/off-ledger" has been used instead.

**Review by amarantha-k — CHANGES_REQUESTED** [review]

**Comment by amarantha-k** [comment]
Noting here for completion - the link to the NFT Metadata blog is proposed and may change.

**Comment by DennisDawson** [comment]
Note that the accompanying blog post is staged here:
https://github.com/ripple/dev-blog/blob/DennisDawson-nft_storage/content/2023/nft-payload-storage.md

**Review by amarantha-k — APPROVED** [review]

**Pull request merged**